src/ui/pages/ManageWallets/WalletAccount/WalletAccount.tsx

```diff
@@ -159,13 +177,18 @@ export function WalletAccount() {
     isLoading,
     refetch: refetchWallet,
   } = useQuery({
-    queryKey: ['wallet/uiGetWalletByAddress', address],
-    queryFn: () => walletPort.request('uiGetWalletByAddress', { address }),
+    queryKey: ['wallet/uiGetWalletByAddress', address, groupId],
+    queryFn: () =>
+      walletPort.request('uiGetWalletByAddress', { address, groupId }),
     useErrorBoundary: true,
   });
+  const { data: walletGroup, isLoading: walletGroupIsLoading } = useQuery({
+    queryKey: ['getWalletGroupByAddress', address],
+    queryFn: () => getWalletGroupByAddress(address),
```

Comment:
Should `useErrorBoundary: true` be added here?

Reply:
🤔  Makes sense, but currently there's a "NotFound" page being used. I have to remember whether this flow was on purpose